core/Piranha.Manager/assets/src/js/piranha.pageedit.js

```diff
@@ -431,7 +431,7 @@ piranha.pageedit = new Vue({
             }
         },
         onExcerptBlur: function (e) {
-            this.excerpt = e.target.innerHTML;
+            this.excerpt = tinyMCE.activeEditor.getContent();
```

Comment:
Since excerpt is configurable between plain text/html, will this work if excerpt is in plain text?

Reply:
Also just realized we need to change it for Post Edit as well!